Tasks/AzureResourceGroupDeploymentV2/make.json

```diff
@@ -1,7 +1,7 @@
 {
     "common": [
         {
-            "module": "../Common/azure-arm-rest",
+            "module": "../Common/azure-arm-rest-v2",
```

Comment:
As mentioned earlier, I am am not sure this is right thing to do

Reply:
same as above.